jetbrains_academy_introduction_to_java.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`.iml` files in general should not be committed. These are internal files used by IntelliJ Idea.
Better to exclude them from versioning by adding a corresponding rule to `.gitignore`